mail_debrand/models/mail_render_mixin.py

```diff
@@ -97,3 +97,10 @@ def _render_template(
             orginal_rendered[key] = self.remove_href_odoo(orginal_rendered[key])
 
         return orginal_rendered
+
+    def _replace_local_links(self, html, base_url=None):
+        message = super()._replace_local_links(html)
+        message = re.sub(
+            r"""(Powered by\s(.*)Odoo</a>)""", "<div>&nbsp;</div>", message
```

Comment:
We replace the "Powered by..." with an empty space. For some templates, the space is ignored and it is removed the grey padding at the bottom part of the message. For this reason it is required the <div><div>.